integration/src/integration/java/pl/allegro/tech/hermes/integration/management/SubscriptionManagementTest.java

```diff
@@ -320,6 +320,26 @@ public void shouldReturnNoDataStatusWhenGraphiteRespondsWithAnError() {
         assertThat(subscriptionHealth).isEqualTo(SubscriptionHealth.NO_DATA);
     }
 
+    @Test
+    public void shouldCreateSubscriptionWithSuspendedStatus() {
```

Comment:
I was wondering wether this test is really necessary. I wanted to be sure that `ACTIVE` is not set as default and overwrites the value that comes from the request.